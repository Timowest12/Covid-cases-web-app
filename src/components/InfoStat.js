import React from 'react';

const InfoStat = (props) => (
  <div className="infoStatsBlock">
    <div className="infoStat">
      <div className="infoStatLeft">{props.metric}</div>
      <div className="infoStatright">{props.amount}</div>
    </div>
  </div>

);

export default InfoStat;
