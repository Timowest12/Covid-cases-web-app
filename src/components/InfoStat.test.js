import React from 'react';
import renderer from 'react-test-renderer';
import InfoStat from './InfoStat';


it('test if infostat component renders correctly', () => {
  const tree = renderer.create(<InfoStat  />).toJSON();
  expect(tree).toMatchSnapshot();
});