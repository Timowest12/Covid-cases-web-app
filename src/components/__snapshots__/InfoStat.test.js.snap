// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div
  className="infoStatsBlock"
>
  <div
    className="infoStat"
  >
    <div
      className="infoStatLeft"
    />
    <div
      className="infoStatright"
    />
  </div>
</div>
`;

exports[`test if infostat component renders correctly 1`] = `
<div
  className="infoStatsBlock"
>
  <div
    className="infoStat"
  >
    <div
      className="infoStatLeft"
    />
    <div
      className="infoStatright"
    />
  </div>
</div>
`;
